scipy-1.0/paper.tex

```diff
@@ -503,6 +503,12 @@ \subsection*{SciPy Today}
 \num{600000}~lines of code organized in 16~subpackages. The development
 team and community currently interact and operate primarily
 on GitHub, an online version control and task management platform.
+According to the practice of continuous integration\cite{silver2017collaborative},
+all proposed contributions to SciPy via GitHub are temporarily integrated with the
+master branch of the library
+before the test suite is run, and all tests must pass before the contribution
```

Comment:
If the reader is not familiar with "continuous integration", they might not be familiar with "test suite".

Reply:
Instead of moving that part, how about adding a bit of detail to the figure caption. After all, if the reader is not familiar with continuous integration, they may not be familiar with a bunch of other things mentioned in the figure that haven't been described yet, like Cython, BLAS/LAPACK, and GitHub. I'll suggest some text.

Reply:
How about leaving the definitions where they make sense and referring the reader to them here? I don't think we need to do _exactly_ what the reviewer says; I think this would adequately address the comment without creating new problems.

Caption of Figure 2:
```
Major milestones from SciPy's initial release in 2001 to the release of SciPy~1.0 in 2017. Note that SciKits and GitHub have been introduced in the ``Background'' section; more information about Cython and SciPy subpackages (e.g. \texttt{scipy.sparse}) is available in ``Architecture and implementation choices'', BLAS/LAPACK support is detailed in ``Key technical improvements'', and continuous integration is discussed in ``Test and benchmark suite''.
```

I wish we could use section/subsection references....

Reply:
I'd rather just do what I did, but fair enough. I'll do this--can someone get the latexdiff thing working with references + bibtex in the meantime?